must-gather/collection-scripts/gather_nodes

```diff
@@ -88,5 +92,7 @@ done
 wait "${ADM_PIDS[@]}"
 
 oc delete -f $DAEMONSET_MANIFEST
+oc delete -f $CLUSTER_ROLE_BINDING_PODS_READER
+oc delete -f $CLUSTER_ROLE_PODS_READER
```

Comment:
for a future PR, we will need to make sure we don't leak kubernetes objects if must-gather fails/it's interrupted

Reply:
yeah I was going to add it in this one but I though it would be better to split it in another one.